src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 57 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt && testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e
```powershell
At line:1 char:81
+ ...  > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt && testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e
```powershell
At line:1 char:81
+ ...  > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt || testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e
```powershell
At line:1 char:81
+ ...  > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt || testexe -returncode 1 > /private/tmp/5c8b458f-0150-4e
```powershell
At line:1 char:81
+ ...  > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement '(testexe -returncode 0 && testexe -returncode 1) > /private/tmp/5c8b458f-0150-4e56-96a4-81f2abbbab7a/3.txt'
```powershell
At line:1 char:24
+ (testexe -returncode 0 && testexe -returncode 1) > /private/tmp/5c8b4 ...
+                        ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```